okhttp/src/main/java/okhttp3/HttpUrl.java

```diff
@@ -1257,6 +1257,9 @@ public HttpUrl build() {
     }
 
     @Override public String toString() {
+      if (scheme == null) throw new IllegalStateException("scheme == null");
```

Comment:
I'm not convinced by this change, an incomplete HttpUrl.Builder is still valid, and should have a toString e.g. if debugging. #tioli

Reply:
unrelated to this change, since this change just changes the error message, but, yeah, maybe toString should have some understandable message in a string here.